### PR TITLE
Add Frappe and ERPNext icons

### LINKED ISSUE #9242: ERPNext Icon
### Brand Name

Erpnext

### Website

https://erpnext.com/

### Popularity Metric

Similarweb rank: 93,718
GitHub stars: 14.5K 

https://www.similarweb.com/website/erpnext.com/#overview

### Official Resources for Icon and Color

svg: https://raw.githubusercontent.com/frappe/erpnext/c1dd06065b6d1aad97d909362a7d71a39fd99608/erpnext/public/images/erpnext-logo.svg

color: #0089FF

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by walstanb**
erpnext

frappe

**Issue:** closes #9242 and #9243 

**Similarweb rank:**
ERPNext: **93,718** See: https://www.similarweb.com/website/erpnext.com/#overview
Frappe: **480,000** See: https://www.similarweb.com/website/frappe.io/#overview

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
 * Extracted the logo from[ Frappe](https://raw.githubusercontent.com/frappe/frappe/59e1db7bb58e155744f9a2e7912fd233f9fac981/frappe/public/images/frappe-framework-logo.svg) & [ERPNext](https://raw.githubusercontent.com/frappe/erpnext/c1dd06065b6d1aad97d909362a7d71a39fd99608/erpnext/public/images/erpnext-logo.svg) Github Repos
 * Color: Taken from logo `#0089FF`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3830,6 +3830,11 @@
             "hex": "A90533",
             "source": "https://github.com/erlang/erlide_eclipse/blob/99d1d61fde8e32ef1630ca0e1b05a6822b3d6489/meta/media/erlang-logo.svg"
         },
+        {
+            "title": "ERPNext",
+            "hex": "0089FF",
+            "source": "https://raw.githubusercontent.com/frappe/erpnext/c1dd06065b6d1aad97d909362a7d71a39fd99608/erpnext/public/images/erpnext-logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:3836]:
```suggestion
            "source": "https://github.com/frappe/erpnext/blob/924911e74317f95a59f29e9410d4f141020a0411/erpnext/public/images/erpnext-logo.svg"
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4469,6 +4474,11 @@
             "hex": "EC6237",
             "source": "https://www.franprix.fr"
         },
+        {
+            "title": "Frappe",
+            "hex": "0089FF",
+            "source": "https://raw.githubusercontent.com/frappe/frappe/59e1db7bb58e155744f9a2e7912fd233f9fac981/frappe/public/images/frappe-framework-logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:4480]:
```suggestion
            "source": "https://github.com/frappe/frappe/blob/1331fa6f721122805d15894fa3bd9eb90dccdca2/frappe/public/images/frappe-framework-logo.svg"
```

File: icons/erpnext.svg
Comment (adamrusted) [inline icons/erpnext.svg:1]: Spotting [some slight differences](https://github.com/simple-icons/simple-icons/assets/1830867/2eaed456-84bf-4e33-8a25-9009812e9f8e) with the source in blue, and the PR in black. Looks like you've optimised this to 1 decimal of precision. Could you regenerate to 3 please? 🙏

File: icons/frappe.svg
Comment (adamrusted) [inline icons/frappe.svg:1]: Same as with ERPNext, I'm seeing [minor differences](https://github.com/simple-icons/simple-icons/assets/1830867/f10710d2-e3c0-4d9e-b0ac-a377e9cf31c9). Again, seems to be down to the icon being saved with only one decimal of precision.

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by PeterShaggyNoble** [comment]
Any update on these, @walstanb?

File: icons/erpnext.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ERPNext</title><path d="M0 2.9C0 1.3 1.3 0 2.9 0h18.2C22.7 0 24 1.3 24 2.9v18.2c0 1.6-1.3 2.9-2.9 2.9H2.9C1.3 24 0 22.7 0 21.1V2.9zm15.8 5h.4V5.8H7.9v2.1h7.9zm-5.3 8v-3h5.2v-2.1H7.9V18h8.3v-2.1h-5.7z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/erpnext.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ERPNext</title><path d="M2.88 0C1.29 0 0 1.29 0 2.88v18.24C0 22.71 1.29 24 2.88 24h18.24c1.59 0 2.88-1.29 2.88-2.88V2.88C24 1.29 22.71 0 21.12 0Zm5.04 5.76h8.254v2.146H7.92Zm0 5.033h7.85v2.146h-5.233v2.954h5.703v2.146H7.92Z"/></svg>
```

File: icons/frappe.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Frappe</title><path d="M0 2.9C0 1.3 1.3 0 2.9 0h18.2C22.7 0 24 1.3 24 2.9v18.2c0 1.6-1.3 2.9-2.9 2.9H2.9C1.3 24 0 22.7 0 21.1V2.9zM16.4 6h-8v2.1h8V6zm-8 5.4V18H11v-4.5h4.9v-2.1H8.4z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/frappe.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Frappe</title><path d="M2.88 0C1.29 0 0 1.29 0 2.88v18.24C0 22.71 1.29 24 2.88 24h18.24c1.59 0 2.88-1.29 2.88-2.88V2.88C24 1.29 22.71 0 21.12 0H2.88zM8.4 6h8.024v2.111H8.4V6zm0 5.385h7.49v2.113h-4.93V18H8.4v-6.615z"/></svg>
```

**Review by PeterShaggyNoble — DISMISSED** [review]

**Review by adamrusted — APPROVED** [review]

**Comment by walstanb** [comment]
Thanks @PeterShaggyNoble I wasn't getting a chance to make changes.

**Pull request merged**